litestar/app.py

```diff
@@ -566,6 +581,20 @@ async def lifespan(self) -> AsyncGenerator[None, None]:
 
             yield
 
+    @contextmanager
+    def server_lifespan(self) -> Generator[None, None, None]:
```

Comment:
Can this be private for now or just moved to the CLI? I feel like we shouldn't commit to this API to soon

Reply:
sounds like a candidate for one of those experimental features sortve